modules/permutiveRtdProvider.md

```diff
@@ -31,39 +32,117 @@ pbjs.setConfig({
-
-## Parameters
+The parameters below provide configurability for general behaviours of the RTD submodule,
+as well as enabling settings for specific use cases mentioned above (e.g. acbidders).
 
 | Name                   | Type     | Description                                                                                   | Default |
-| ---------------------- | -------- | --------------------------------------------------------------------------------------------- | ------- |
+|------------------------|----------|-----------------------------------------------------------------------------------------------|---------|
 | name                   | String   | This should always be `permutive`                                                             | -       |
 | waitForIt              | Boolean  | Should be `true` if there's an `auctionDelay` defined (optional)                              | `false` |
 | params                 | Object   |                                                                                               | -       |
 | params.acBidders       | String[] | An array of bidders which should receive AC cohorts.                                          | `[]`    |
 | params.maxSegs         | Integer  | Maximum number of cohorts to be included in either the `permutive` or `p_standard` key-value. | `500`   |
 | params.transformations | Object[] | An array of configurations for ORTB2 user data transformations                                |         |
 
-### The `transformations` parameter
+##### The `transformations` parameter
 
 This array contains configurations for transformations we'll apply to the Permutive object in the ORTB2 `user.data` array. The results of these transformations will be appended to the `user.data` array that's attached to ORTB2 bid requests.
 
-#### Supported transformations
+##### Supported transformations
 
 | Name           | ID  | Config structure                                  | Description                                                                          |
-| -------------- | --- | ------------------------------------------------- | ------------------------------------------------------------------------------------ |
+|----------------|-----|---------------------------------------------------|--------------------------------------------------------------------------------------|
 | IAB taxonomies | iab | { segtax: number, iabIds: Object<number, number>} | Transform segment IDs from Permutive to IAB (note: alpha version, subject to change) |
+
+#### Context
+
+Permutive is not listed as a TCF vendor as all data collection is on behalf of the publisher and based on consent the publisher has received from the user.
+This consent is provided to Permutive on the SDK level, rather than through the TCF framework.
```

Comment:
Can you clarify what sdk means here?

Reply:
I have reworded - hopefully a bit clearer:
```
Rather than through the TCF framework, this consent is provided to Permutive when the user gives the relevant permissions on the publisher website which allow the Permutive SDK to run.
```